diff --git a/dist/src/SlackWebSocket.js b/dist/src/SlackWebSocket.js
index 1e38e34650c34b8cd78e9fc7e76ae3b611a16c91..e8ba761ea80cdec116ca0538a9564a203e4f0fbc 100644
--- a/dist/src/SlackWebSocket.js
+++ b/dist/src/SlackWebSocket.js
@@ -35,7 +35,7 @@ var __importStar = (this && this.__importStar) || (function () {
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.SlackWebSocket = exports.WS_READY_STATES = void 0;
 const node_diagnostics_channel_1 = require("node:diagnostics_channel");
-const undici_1 = require("undici");
+const undici_1 = require("undici/index.js");
 const errors_1 = require("./errors");
 const logger_1 = __importStar(require("./logger"));
 exports.WS_READY_STATES = ['CONNECTING', 'OPEN', 'CLOSING', 'CLOSED'];
diff --git a/dist/src/SocketModeClient.js b/dist/src/SocketModeClient.js
index 1809f52fba621ab556098d4b03c04fee898736e5..e306e2ec43014c50cf7d9f6cb39eead442c20f07 100644
--- a/dist/src/SocketModeClient.js
+++ b/dist/src/SocketModeClient.js
@@ -39,7 +39,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.SocketModeClient = void 0;
 const web_api_1 = require("@slack/web-api");
 const eventemitter3_1 = require("eventemitter3");
-const undici_1 = require("undici");
+const undici_1 = require("undici/index.js");
 const package_json_1 = __importDefault(require("../package.json"));
 const errors_1 = require("./errors");
 const logger_1 = __importStar(require("./logger"));
